### PR TITLE
Fc xxx apprentice content

### PR DESCRIPTION
**Pull request by desderry**

### CODE REVIEW

File: src/SFA.DAS.Campaign.Web/Controllers/RegisterInterestController.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                    FirstName = registerInterest.FirstName,
+                    LastName = registerInterest.LastName,
+                    Email = registerInterest.EmailAddress,
+                    CookieId = registerInterest.ReturnUrl,
+                    RouteId = registerInterest.ReturnUrl
+                };
+
+                await _userDataCollection.StoreUserData(userData);
+
+                return Redirect(registerInterest.ReturnUrl + $"#{ModalIdConsts.RegisterThanksId}");
```
Comment (dashton82) [inline src/SFA.DAS.Campaign.Web/Controllers/RegisterInterestController.cs]:
```suggestion
                return Redirect($"{registerInterest.ReturnUrl}#{ModalIdConsts.RegisterThanksId}");
```

File: src/SFA.DAS.Campaign.Web/ViewComponents/Hero-Heading/HeroHeadingViewComponent.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        public async Task<IViewComponentResult> InvokeAsync(HeroHeadingType type, string caption, string classes, IHtmlContent content)
+        {
+           
+            return View("Default", new HeroHeadingViewModel(type, caption, classes, content));
+        }
+    }
+
+    public class HeroHeadingViewModel
+    {
+        public HeroHeadingViewModel(HeroHeadingType type, string caption, string classes, IHtmlContent content)
```
Comment (dashton82) [inline src/SFA.DAS.Campaign.Web/ViewComponents/Hero-Heading/HeroHeadingViewComponent.cs]: Should this be in a separate file? Also is it worth changing Caption, Class, Type and Content to be `internal set`

File: src/SFA.DAS.Campaign.Web/ViewComponents/Modal/ModalViewComponent.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                Id = id,
+                Type = type
+            };
+
+
+            return View("Modal", modalVM);
+        }
+    }
+
+    public class ModalViewModel
```
Comment (dashton82) [inline src/SFA.DAS.Campaign.Web/ViewComponents/Modal/ModalViewComponent.cs]: Separate file?

File: src/SFA.DAS.Campaign.Web/Views/Apprentice/MonitorYourProgress.cshtml [hunk 4]
```diff
@@ -0,0 +1,56 @@
+﻿@{
+    ViewBag.Title = "MONITORING AND PROGRESS";
```
Comment (dashton82) [inline src/SFA.DAS.Campaign.Web/Views/Apprentice/MonitorYourProgress.cshtml]: Should it be 'Monitoring your Progress' ? and lowercase?

File: src/SFA.DAS.Campaign.Web/Views/Employer/TheRightApprenticeship.cshtml [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+</p>
+<p>
+    Once an apprentice finishes their training they will be assessed by an independent apprentice assessment organisation.
+</p>
+<p>    ##table - to be added ##</p>
+
+<p>
+    Apprenticeships are being developed, approved and added to the list all the time.  Keep a lookout for new apprenticeships that have been added and make sure that you find the right apprenticeship to meet your business needs.
+</p>
+FURTHER READING (Links):
```
Comment (dashton82) [inline src/SFA.DAS.Campaign.Web/Views/Employer/TheRightApprenticeship.cshtml]: Maybe take these out for now

File: src/SFA.DAS.Campaign.Web/Views/Employer/WhatIsAnApprenticeship.cshtml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+﻿@model object
```
Comment (dashton82) [inline src/SFA.DAS.Campaign.Web/Views/Employer/WhatIsAnApprenticeship.cshtml]: Does this page need removing?

File: src/SFA.DAS.Campaign.Web/Views/Shared/Components/Modal/Modal.cshtml [hunk 7]
```diff
@@ -0,0 +1,31 @@
+﻿@using SFA.DAS.Campaign.Web.ViewComponents.Modal
+@model SFA.DAS.Campaign.Web.ViewComponents.Modal.ModalViewModel
+
+<div class="modal" id="@Model.Id" aria-hidden="true">
+
+    <div class="modal-dialog">
+        <div class="modal-header">
+            <a href="#" class="close">
+                <svg width="32" height="32" viewBox="0 0 32 32" class="">
+                    <rect class="modal--icon-close" x="1.58691" y="0.0893555" width="16.0371" height="2.16583" transform="rotate(45 1.58691 0.0893555)">
```
Comment (dashton82) [inline src/SFA.DAS.Campaign.Web/Views/Shared/Components/Modal/Modal.cshtml]: Do these inline styles need to be moved to the CDN?

File: src/SFA.DAS.Campaign.Web/ViewComponents/Hero-Heading/HeroHeadingViewComponent.cs (hunk 2 above)
Comment (desderry) [reply src/SFA.DAS.Campaign.Web/ViewComponents/Hero-Heading/HeroHeadingViewComponent.cs]: moved but cannot be internal set as access is required from  views

File: src/SFA.DAS.Campaign.Web/ViewComponents/Modal/ModalViewComponent.cs (hunk 3 above)
Comment (desderry) [reply src/SFA.DAS.Campaign.Web/ViewComponents/Modal/ModalViewComponent.cs]: done

File: src/SFA.DAS.Campaign.Web/ViewComponents/Hero-Heading/HeroHeadingViewComponent.cs (hunk 2 above)
Comment (desderry) [reply src/SFA.DAS.Campaign.Web/ViewComponents/Hero-Heading/HeroHeadingViewComponent.cs]: setters are now changed to internal where appropriate

**Pull request merged**